docs/multi-homing.md

```diff
@@ -22,8 +22,11 @@ configurations are defined in a CRD named `NetworkAttachmentDefinition`.
 Below you will find example attachment configurations for each of the current
 topologies OVN-K allows for secondary networks.
 
-**NOTE**: currently, all the secondary networks **only** allow for east/west
-traffic.
+**NOTE**:
+- currently, all the secondary networks **only** allow for east/west traffic.
```

Comment:
nit: I would have this comment on each network topology section as this would no longer hold true when localnet is added

Reply:
IMO, either localnet lands first, and this has to be re-written, or this lands first, and localnet has to update the README. Something they need to do any way, right ?

Reply:
You can always do something so that it is easier or more difficult to update in the future and we already know what's coming. But fair enough.

Reply:
OK, resolving.